canlink/canclient.go

```diff
@@ -95,12 +97,14 @@ func (c *CanClient) Read(ctx context.Context, msgsToRead ...generated.Message) (
 			return nil, nil
 		case frame := <-c.rxChan:
 			msg, err := c.md.UnmarshalFrame(frame)
-			if err != nil && !isIdNotInDatabaseError(err) {
-				return nil, errors.Wrap(err, "unmarshal frame")
-			} else if isIdNotInDatabaseError(err) {
-				c.l.Debug("found a message we do not recognize")
-				// Here we have simply read a can frame that we do not know how to unmarshal, continue to next frame.
-				continue
+			if err != nil {
```

Comment:
@samparent97 @langei see [this comment](https://github.com/macformula/hil/pull/99#discussion_r1893235963)